packages/jellyfish-api-core/__tests__/category/loan/takeLoan.test.ts

```diff
@@ -753,29 +754,32 @@ describe('takeloan failed', () => {
     await expect(promise).rejects.toThrow('No live fixed prices for META/USD')
   })
 
-  it('should not takeLoan when DFI collateral value less than 50%', async () => {
+  it('should not takeLoan when DFI collateral value less than 50% of loan value', async () => {
```

Comment:
`50% of loan value` is not exactly termed. maybe
```suggestion
  it('should not takeLoan when DFI collateral value less than 50% of the minimum required collateral value', async () => {
```